complaints/models.py

```diff
@@ -34,7 +34,40 @@ class Complaint(models.Model):
         (CATEGORY_3, "Sophos Firewall"),
     )
 
+    KRISHNA = "K"
+    RAM = "R"
+    SHANKAR = "S"
+    GANDHI = "G"
+    BUDH = "B"
+    VYAS = "V"
+    GAUTAM = "GT"
+    VALMIKI = "VL"
```

Comment:
acronym for valmiki is VM

Reply:
are the others ok?

Reply:
I am not sure about Meera and Malviya. Ask Kavya di for those.Rest other are fine.